src/app/devices/sensor-details/sensor-detail.component.test.ts

```diff
@@ -70,7 +71,7 @@ describe('sensor-detail', () => {
     });
 
     it('should not make get request when creating new sensor', () => {
-        route = new ActivatedRouteMock();
+        route = new ActivatedRouteMock(ROUTING.CREATE);
```

Comment:
In my opinion in tests will be better to use 'magic values' to prevent importing a lot of constants.